ArduPlane/altitude.cpp

```diff
@@ -352,10 +352,19 @@ int32_t Plane::calc_altitude_error_cm(void)
 }
 
 /*
-  check for FBWB_min_altitude_cm violation
+  check for min and max altitude: FBWB_min_altitude_cm and FBWB_max_altitude_m violation
  */
-void Plane::check_fbwb_minimum_altitude(void)
+void Plane::check_fbwb_altitude(void)//check_fbwb_minimum_altitude
 {
+    //check for max
+    if (g.FBWB_max_altitude_m != 0)
```

Comment:
test should be >0...often people use -1 to disable things, irrespective of what the metadata says

Reply:
should also check its larger than alt min

Reply:
mb larger than alt min and some const ?
Is 1 m correct max alt? 5 m? 10 m?
I wonna do some like:
 **if** (g.FBWB_max_altitude_m*1000 > g.FBWB_min_altitude_cm **&&** g.FBWB_max_altitude_m > **10**)